lib/stats/AdditionalStatsPlugin.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+const pluginName = "additionalStatsPlugin";
```

Comment:
Could you capitalize all plugin names please so they match the classname and the filename?